docs/source/quick_start.md

```diff
@@ -6,98 +6,38 @@
 - Atlas A2 Training series (Atlas 800T A2, Atlas 900 A2 PoD, Atlas 200T A2 Box16, Atlas 300T A2)
 - Atlas 800I A2 Inference series (Atlas 800I A2)
 
-<!-- TODO(yikun): replace "Prepare Environment" and "Installation" with "Running with vllm-ascend container image" -->
-
-### Prepare Environment
-
-You can use the container image directly with one line command:
-
-```bash
-# Update DEVICE according to your device (/dev/davinci[0-7])
-DEVICE=/dev/davinci7
-IMAGE=quay.io/ascend/cann:8.0.rc3.beta1-910b-ubuntu22.04-py3.10
-docker run \
-    --name vllm-ascend-env --device $DEVICE \
-    --device /dev/davinci_manager --device /dev/devmm_svm --device /dev/hisi_hdc \
-    -v /usr/local/dcmi:/usr/local/dcmi -v /usr/local/bin/npu-smi:/usr/local/bin/npu-smi \
-    -v /usr/local/Ascend/driver/lib64/:/usr/local/Ascend/driver/lib64/ \
-    -v /usr/local/Ascend/driver/version.info:/usr/local/Ascend/driver/version.info \
-    -v /etc/ascend_install.info:/etc/ascend_install.info \
-    -v /root/.cache:/root/.cache \
-    -it --rm $IMAGE bash
-```
-
-You can verify by running below commands in above container shell:
+## Setup environment using container
 
 ```bash
-npu-smi info
-```
-
-You will see following message:
-
-```
-+-------------------------------------------------------------------------------------------+
-| npu-smi 23.0.2              Version: 23.0.2                                               |
-+----------------------+---------------+----------------------------------------------------+
-| NPU   Name           | Health        | Power(W)    Temp(C)           Hugepages-Usage(page)|
-| Chip                 | Bus-Id        | AICore(%)   Memory-Usage(MB)  HBM-Usage(MB)        |
-+======================+===============+====================================================+
-| 0     xxx            | OK            | 0.0         40                0    / 0             |
-| 0                    | 0000:C1:00.0  | 0           882  / 15169      0    / 32768         |
-+======================+===============+====================================================+
-```
-
-
-## Installation
+# You can change version a suitable one base on your requirement, e.g. main
+export IMAGE=ghcr.io/vllm-project/vllm-ascend:v0.7.1.rc1
```

Comment:
We'd better use this: https://github.com/vllm-project/vllm-ascend/pull/90

| branch/tag | vllm_version | vllm_ascend_version|
|----|----|----|
| main | main | main |
| v0.7.1-dev | v0.7.1 | v0.7.1-dev |
| v0.7.1rc1 | v0.7.1 | v0.7.1rc1 |